src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitWorkerService.java

```diff
@@ -48,6 +49,9 @@ public class VoltageInitWorkerService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(VoltageInitWorkerService.class);
 
+    private static final String ERROR_KEY = "error";
```

Comment:
I never seen `KEY` at the end of constants in Gridsuite, maybe remove it ?

Reply:
Actually, the key is also a label in the front. It could be in caps "ERROR" or "FAILED"

Reply:
My comment was about the value, not the variable name. But give it up. It should be set in caps in the front

Reply:
Agreed to remove the 'KEY'

Reply:
fixed